mariner-installer/build/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+echo "BRANCH: $BRANCH"
+cd $WORKSPACE/mariner-installer
+
+. /etc/os-release
```

Comment:
Okay, so let's figure out what we want to do, and I'll submit a separate PR to fixup takora to do that, then apply that same change here.

`ceph-deploy` checks if `$DIST` is in a static list: https://github.com/ceph/ceph-build/blob/master/ceph-deploy/build/build#L35-L42

`ceph-build` simply calls two scripts, and has the irrelevant script exit immediately:
https://github.com/ceph/ceph-build/blob/master/ceph-build/build/build_deb#L5-L7
https://github.com/ceph/ceph-build/blob/master/ceph-build/build/build_rpm#L4-L6

The "original" way before 3bd7177 was as it is in `ice-setup`, checking $DIST as well: https://github.com/ceph/ceph-build/blob/master/ice-setup/build/build#L8-L12

`radosgw-agent` uses a different permutation of all this: https://github.com/ceph/ceph-build/blob/master/radosgw-agent/build/build#L25

Voting is now open.  :smiley: